services/webnn/ort/graph_builder_ort.cc

```diff
@@ -337,6 +337,40 @@ GraphBuilderOrt::CreateOrReshapeBias(const std::optional<uint32_t>& bias_id,
   return bias;
 }
 
+std::string GraphBuilderOrt::ConvertRznToZrn(std::string_view weight_or_bias) {
+  uint32_t num_gates = 3;
```

Comment:
Nit: this variable seems to be unnecessary, see comments below.

Reply:
> Nit: this variable seems to be unnecessary, see comments below.

Yes, this variable was created when I was trying to implement the layout parameters of gru and lstm together, but since the code is a bit ugly, I wrote it separately first.